Business Finder/AppDelegate.h

```diff
@@ -8,10 +8,11 @@
 
 #import <UIKit/UIKit.h>
 #import <CoreData/CoreData.h>
+#import "Model.h"
 
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
 
 @property (strong, nonatomic) UIWindow *window;
-
+@property (strong, nonatomic) Model *model;
```

Comment:
I removed the `model` property from the `AppDelegate`, and refactored things so we just pass around the relevant data between view controllers.